test/tedious/tedious.js

```diff
@@ -229,7 +229,8 @@ describe('tedious', () => {
 
   describe('connection errors', function () {
     it('login failed', done => TESTS['login failed'](done, /Login failed for user '(.*)'/))
-    it('timeout', done => TESTS.timeout(done, /Failed to connect to 10.0.0.1:1433 in 1000ms/))
+    // call(this) to enable the test to skip itself.
+    it('timeout', function (done) { TESTS.timeout.call(this, done, /Failed to connect to 10.0.0.1:1433 in 1000ms/) })
```

Comment:
Does this work, I didn't think you could bind new `this` context to arrow functions...

Reply:
this works because TESTS.timeout is not an arrow function, but a method definition: `{ timeout(args) { body } }` rather than `{ timeout: (args) => { body } }`. see output in the node console of:

```js
t = { meth() { console.log(this) }, arr: () => { console.log(this) } }
t.meth.call('custom this')
t.arr.call('custom this')
```

arr has the globalThis anyway, but meth has 'custom this' as its this.